### ISSUE TITLE
Autoloader matches the wrong class

### ISSUE THREAD

**Issue by tomasfejfar** [pear/pear-core-minimal#1]
```
Warning: Ambiguous class resolution, "PEAR_ErrorStack" was found in both "vendor/pear/pear-core-minimal/src/PEAR/ErrorStack.php" and "vendor/pear/pear-core-minimal/src/PEAR/ErrorStack5.php", the first will be used.
```

**Comment by alecpl** [comment]
This is fixed in 1.10.0 and can be closed. BTW, please update list of files in README.rst file.

**Comment by cweiske** [comment]
done.
